domain/network/state/linklayer_merge_test.go

```diff
@@ -838,7 +1070,35 @@ func (s *mergeLinkLayerSuite) addProviderIPAddress(
 	`, providerID, addressUUID)
 }
 
-// createNetInterface creates a network.NetInterface for testing.
+// addSpace adds a space to the database and returns its UUID.
+func (s *mergeLinkLayerSuite) addSpace(c *tc.C, name string) string {
```

Comment:
This helper exists in the `linkLayerBaseSuite`, same with `addSubnet`. Do we need two versions of it?